LockboxXCUITests/LockBoxScreenGraph.swift

```diff
@@ -100,6 +105,18 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
     map.addScreenState(Screen.LockboxMainPage) { screenState in
         screenState.tap(app.buttons["settings.button"], to: Screen.SettingsMenu)
         screenState.tap(app.buttons["sorting.button"], to: Screen.SortEntriesMenu)
+        screenState.tap(app.tables.cells.staticTexts.firstMatch, to: Screen.EntryDetails)
+    }
+
+    map.addScreenState(Screen.EntryDetails) { screenState in
+        screenState.gesture(forAction: Action.RevealPassword) { userState in
+            app.buttons["reveal.button"].tap()
+        }
+
+        screenState.gesture(forAction: Action.OpenWebsite) { userState in
+            app.cells["Web Address"].press(forDuration: 1)
```

Comment:
This needs an `accessibilityIdentifier`.

Reply:
If I'm not wrong, that's already an identifier:

```
Cell, 0x600000197f80, traits: 8589934592, {{0.0, 151.0}, {375.0, 58.0}}, identifier: 'Web Address', label: 'Web address: double tap to open in browser https://accounts.google.com'
```

Reply:
It doesn't follow the pattern of our other `accessibilityIdentifier`s though, you're right.... I can fix the `ItemDetail` accessibilityIDs on this branch if that's an acceptable solution to this issue.

Reply:
Oh I see, yup, the pattern is different. So, we have the same issue with the `accessibilityIdentifier`s  in `settingsView` , please see [here](https://github.com/mozilla-lockbox/lockbox-ios/blob/master/LockboxXCUITests/LockBoxScreenGraph.swift#L119). Identifiers are set but they name used is the `cellConfiguration.text`, and do not follow the pattern either.
I was trying to fix them all but have not found how to do it correctly yet. Only got the `Firefox Lockbox` (navigationBar) 
If you want to fix them in this branch is great, but if you prefer I can try to do it with some guidance, what is faster for you. Both ways I will learn how to do it so that I can do it by myself next time.
Thank you!!

Reply:
I'd be really happy for you to be able to fix them for yourself.

The quickest way I have found is to start with the text and trace it back to the view itself. e.g. 

for a view with the label: `Example View`, 

 * searching all files in Xcode for the text `Example View` reveals the declaration of `AppConstants.labelExampleView`
 * searching all files in Xcode for the `labelExampleView` reveals where it is used to set the label text. This is likely where the label is constructed.
 * where it is constructed, you can add your own `accessibilityIdentifier`.

For how to set the identifier: pick the pattern that existing views have used and use that — but I don't think this is something you're having problems with.

Reply:
Thanks for your suggestion @jhugman.
Finally I was able to find a way to add the `accessibilityIdentifier`s for the Settings options and also for the Item Detail options. Not sure if that is correct or there is a simplier way. 
@sashei, @jhugman Could you please let me know how it looks now? 
Thanks!

Reply:
I think it'd be great if @isabelrios would take ownership of the `accessibilityIdentifier`s. Go with whatever pattern that makes sense (i.e. follow what is already there), but make it different from the label itself.

Reply:
Please ping either sasha or I on how to find where to add accessibility identifiers if you need help.